pages/common/postmap.md

```diff
@@ -0,0 +1,24 @@
+# postmap
+
+> Create or query one or more Postfix lookup tables, or update an existing one.
+> More information: <https://manned.org/postmap>.
+
+- Search specified maps for the key and remove one entry per map:
```

Comment:
```suggestion
- Search specific maps for a key and remove one entry per map (output stream: `stdout`):
```

```suggestion
- Search specific maps for a key and remove one entry per map (output stream: `stderr`):
```

Optional fix: `specific` may sound better.
Optional fix: add the correct output stream name explicitly.